src/Edit.tsx

```diff
@@ -218,16 +218,16 @@ const AppNoError = ({
 
   const scrollToDefRef = useRef<ScrollToDef | undefined>(undefined);
   const scrollToTypeDefRef = useRef<ScrollToDef | undefined>(undefined);
-
+  const defs = p.module.defs
+    .sort((a, b) => cmpName(a.name, b.name))
+    .map((d) => d.name.baseName);
   return (
     <div className="grid h-screen grid-cols-[18rem_auto_20rem]">
       <div className="h-full overflow-hidden">
         <Sidebar
           initialMode="T&D"
           prog={{
-            defs: p.module.defs
-              .sort((a, b) => cmpName(a.name, b.name))
-              .map((d) => d.name.baseName),
+            defs: defs,
```

Comment:
Not important, but I'm a bit surprised that ESLint doesn't recommend simplifying this to `defs,`.

Reply:
Right, I will fix that.

Reply:
Fixed.